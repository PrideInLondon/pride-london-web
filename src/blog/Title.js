import React from 'react'
import PropTypes from 'prop-types'
import { StyledTitle } from './Title.styles'

const Title = ({ children, isLight, isCentered, className }) => (
  <StyledTitle className={className} isLight={isLight} isCentered={isCentered}>
    {children}
  </StyledTitle>
)

Title.propTypes = {
  children: PropTypes.string.isRequired,
  isLight: PropTypes.bool,
  isCentered: PropTypes.bool,
  className: PropTypes.string,
}

Title.defaultProps = {
  isLight: false,
  isCentered: false,
  className: null,
}

export default Title
