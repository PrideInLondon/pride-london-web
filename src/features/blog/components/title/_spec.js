import React from 'react'
import { shallow } from 'enzyme'
import Title from './'

describe(Title.name, () => {
  it('renders with default values', () => {
    const wrapper = shallow(<Title />)
    expect(wrapper).toMatchSnapshot()
  });
});
