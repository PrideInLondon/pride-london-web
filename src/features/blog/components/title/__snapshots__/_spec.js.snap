// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders with default values 1`] = `
<styles__StyledTitle
  className={null}
  isCentered={false}
  isLight={false}
>
  Hi
</styles__StyledTitle>
`;
